.chloggen/dockerfile_remove_root_group.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: image
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Set Dockerfile's USER_GID as non root. OpenTelemetry Collector releases issue: https://github.com/open-telemetry/opentelemetry-collector-releases/issues/662"
```

Comment:
[nitpick] can you not add the link to the `issues:` field below?

Reply:
no, you can only link issues / PRs in this repo in `issues: `

Reply:
ok 🤷🏼